dev-docs/bidders/equativ.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: bidder
+title: Equativ
+description: Prebid Equativ Bidder Adapter
+biddercode: equativ
+media_types: banner
+tcfeu_supported: true
+coppa_supported: true
+gvl_id: 45
+gpp_supported: true
+schain_supported: true
+usp_supported: true
+userIds: all
+pbjs: true
+pbs: true
+floors_supported: true
+sidebarType: 1
```

Comment:
there are a number of metadata fields not provided. See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

e.g. deals_supported, fpd_supported, prebid_member, etc.

Reply:
Thanks for the link to that doc - I must confess my starting point was the existing documentation for some other bidders, which are older and do not include all the fields.